configure.ac

```diff
@@ -562,6 +576,20 @@ fi
 
 AC_ARG_WITH(unixodbc,
 AS_HELP_STRING([--with-unixodbc=DIR], [build odbc driver against unixODBC in DIR]))
+
+if test "x$with_unixodbc" = "xyes"; then
+	# The user asked for unixodbc support, but didn't tell us
+	# where to look. We'll try to get the installation prefix from
+	# the odbc_config executable, and if we can't, we'll display
+	# an error. We require odbc_config to be in $PATH.
+	ODBC_INC=`odbc_config --include-prefix`
+	test -r "$ODBC_INC/sql.h" || AC_ERROR([sql.h not found])
+	test -r "$ODBC_INC/sqlext.h" || AC_ERROR([sqlext.h not found])
+	with_unixodbc=`odbc_config --prefix`
```

Comment:
That code is only reached with `--with-unixodbc=yes`. The idea is that, if I pass `--with-unixodbc=yes`, then it will try to use `odbc_config` to find `sql.h` and `sqlext.h`. If it succeeds, then it will change `$with_odbc` from `yes` to `/usr/`, and then continue as if I had specified `--with-unixodbc=/usr` in the first place.

Anyway I don't need this patch so I'm fine if you omit it, thanks for all your help.

I'll look into the documentation updates over the weekend.